.github/workflows/check-markdown.yml

```
@@ -0,0 +1,21 @@
+    branches: [main]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    permissions:
+      statuses: write
+    steps:
+      - uses: bmazzarol/bmazzarol/.github/actions/check-markdown@main
+        with:
+          targetMdFiles: |
+            "Cutout*/**/*.md" "Cutout.Docs/**/*.md" "*.md" "!LICENSE.md"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)